### PR TITLE
Added UI to the event-receiver application 

### PR DESCRIPTION
**Pull request by rshaiju**
Following are changes in this PR

- Added UI to the event-receiver application 
- Modified the ReadMe to reflect the changes

### CODE REVIEW

File: sample/Kmd.Logic.Cpr.Events.Receiver/Kmd.Logic.Cpr.Events.Receiver.csproj [hunk 1]
```diff
@@ -1,13 +1,13 @@
-﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+<Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
     <TargetFramework>net5.0</TargetFramework>
-    <UserSecretsId>af370594-5b69-44aa-8fcd-befcdff9cf8b</UserSecretsId>
```
Comment (mlatoszek) [inline sample/Kmd.Logic.Cpr.Events.Receiver/Kmd.Logic.Cpr.Events.Receiver.csproj:5]: We don't need UserSecrets here?

**Review by mlatoszek — APPROVED** [review]

File: global.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "sdk": {
-    "version": "3.1.101",
+    "version": "5.0.202",
```
Comment (mlatoszek) [inline global.json:3]: I believe you need to update build pipieline. This will break the builds as we're installing core 3.1

**Review by mlatoszek — CHANGES_REQUESTED** [review]

File: global.json (hunk 2 above)
Comment (mlatoszek) [reply global.json:3]: + let's remove appveyour as we're using github actions now

File: sample/Kmd.Logic.Cpr.Events.Receiver/Kmd.Logic.Cpr.Events.Receiver.csproj (hunk 1 above)
Comment (rshaiju) [reply sample/Kmd.Logic.Cpr.Events.Receiver/Kmd.Logic.Cpr.Events.Receiver.csproj:5]: No. The application doesn't have any values that it takes from the appsettings except for the optional TenantName which is not a secret

File: global.json (hunk 2 above)
Comment (rshaiju) [reply global.json]: I have removed global.json (expecting that was what you wanted me to do).  Build is passing now

**Review by mlatoszek — APPROVED** [review]

**Pull request merged**